src/NHibernate.Test/TestsContext.cs

```diff
@@ -15,7 +14,7 @@ namespace NHibernate.Test
 #endif
 	public class TestsContext
 	{
-		public static bool ExecutingWithVsTest { get; } =
+		private static bool ExecutingWithVsTest { get; } =
```

Comment:
Because this is now private this file applies only to .NET Core so it can be put into `#if NETCOREAPP2_0` as a whole.

Reply:
Done